nms/app/components/admin/userManagement/UsersSettings.js

```diff
@@ -144,7 +144,12 @@ function UsersSettings(props: Props) {
     <div className={classes.paper}>
       <div className={classes.header}>
         <Text variant="h5">Users</Text>
-        <Button onClick={() => setShowDialog(true)}>Add User</Button>
+        <Button
+          variant="contained"
+          color="primary"
```

Comment:
Does the primary colour needs explicit specification or would this also be used by default?

Reply:
It needs to be specified, the default is `default` see https://v4.mui.com/api/button/#props